evennia/server/profiling/profiler.py

```diff
@@ -0,0 +1,177 @@
+import gc
```

Comment:
I know this is meant as a WIP, but if it should go into Evennia core, this requires a useful and descriptive doc string including usage examples and maybe even links to further reading - specially since the contents is so technical.